auditlog/registry.py

```diff
@@ -148,4 +154,73 @@ def _dispatch_uid(self, signal, model) -> DispatchUID:
         return self.__hash__(), model.__qualname__, signal.__hash__()
 
 
+def _get_model_classes(app_model: str) -> List[ModelBase]:
+    try:
+        try:
+            app_label, model_name = app_model.split(".")
+            return [apps.get_model(app_label, model_name)]
+        except ValueError:
+            return apps.get_app_config(app_model).get_models()
+    except LookupError:
+        return []
+
+
+def _auditlog_register_models(
+    auditlog: AuditlogModelRegistry, models: Iterable[Union[str, Dict[str, Any]]]
+):
+    models = copy.deepcopy(models)
+
+    if not isinstance(models, list) and not isinstance(models, tuple):
+        raise TypeError(f"'models'({type(models)}) is not an list or tuple")
+
+    for model in models:
+        if isinstance(model, str):
+            for model_class in _get_model_classes(model):
+                auditlog.unregister(model_class)
+                auditlog.register(model_class)
+
+        elif isinstance(model, dict):
+            if "model" not in model:
+                raise ValueError("item must contain 'model' key")
+            if "." not in model["model"]:
+                raise ValueError(
+                    "model with options must be in the format <app_name>.<model_name>"
+                )
+
+            model["model"] = _get_model_classes(model["model"])[0]
+            auditlog.unregister(model["model"])
+            auditlog.register(**model)
+        else:
+            raise TypeError("item must be a dict or str")
+
+
+def get_exclude_models():
+    exclude_models: List[ModelBase] = [
+        model
+        for app_model in getattr(settings, "AUDITLOG_EXCLUDE_TRACKING_MODELS", ())
+        + _DEFAULT_EXCLUDE_MODELS
+        for model in _get_model_classes(app_model)
+    ]
+    return exclude_models
+
+
+def auditlog_register(
+    auditlog: AuditlogModelRegistry,
+    include_all_models=False,
+    include_auto_created=False,
```

Comment:
Why do we have these two args here? Do they need just to make the function easier to test?